### PR TITLE
Migrate from CircleCI 1.0 to 2.0

### LINKED ISSUE #122: Move CircleCI from version 1.0 to version 2.0 before August 31, 2018
After August 31, 2018, CircleCI 1.0 will no longer be available for Linux and macOS users. You can find guides for transitioning from 1.0 to 2.0 and a full timeline on planned changes

Official information
https://circleci.com/blog/sunsetting-1-0/

### PR DESCRIPTION
**Pull request by hectorerb**
### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### References

Closes https://github.com/flyve-mdm/ios-inventory-agent/issues/122

### CODE REVIEW

**Review by ajsb85 — APPROVED** [review]

File: ci/scripts/build.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+#   Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/build.sh:3]: 2017-2018

File: ci/scripts/build.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
+# @date      25/08/17
+# @copyright Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/build.sh:22]: 2017-2018

File: ci/scripts/install_ruby.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+#   Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/install_ruby.sh:3]: 2017-2018

File: ci/scripts/install_ruby.sh [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
+# @date      06/06/18
+# @copyright Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/install_ruby.sh:22]: 2017-2018

File: ci/scripts/set_locate.sh [hunk 5]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+#   Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/set_locate.sh:3]: 2017-2018

File: ci/scripts/set_locate.sh [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
+# @date      06/06/18
+# @copyright Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/set_locate.sh:22]: 2017-2018

File: ci/scripts/set_locate.sh [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
```
Comment (ajsb85) [inline ci/scripts/set_locate.sh:20]: Hi, @hectorerb 
Add also your email address

File: ci/scripts/install_ruby.sh [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
```
Comment (ajsb85) [inline ci/scripts/install_ruby.sh:20]: Hi, @hectorerb 
Add also your email address

File: ci/scripts/build.sh [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# FlyveMDMInventoryAgent is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
```
Comment (ajsb85) [inline ci/scripts/build.sh:20]: Hi, @hectorerb 
Add also your email address

**Review by ajsb85 — DISMISSED** [review]

File: ci/scripts/build.sh (hunk 9 above)
Comment (hectorerb) [reply ci/scripts/build.sh:20]: Yes, I must update the header for the entire project

**Review by ajsb85 — DISMISSED** [review]

**Review by rafaelje — APPROVED** [review]
Reviewed

**Pull request merged**